smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -701,16 +700,15 @@ def validateCommand(device, deviceType, command) {
  * doing it for the device types which accept commands
  * @return attribute name of the device type
  */
-def getCapabilityName(type) {
-    switch(type) {
-		case "switches":
+def getCapabilityName(device) {
+    if (switches.find{it.id == device})
 			return "Switch"
-		case "alarms":
+		if (alarms.find{it.id == device})
 			return "Alarm"
-		case "locks":
+		if (locks.find{it.id == device})
 			return "Lock"
 		default:
```

Comment:
I think we will need get rid of the `default` case here, since it is not a switch case anymore.